packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -58,6 +60,16 @@ const deserializer = (message: any) => {
   }
 }
 
+type connectRequestedActionWithState = [
```

Comment:
```suggestion
type ConnectRequestedActionWithState = [
```
Types should always be PascalCase